api/src/main/java/org/apache/iceberg/expressions/ExpressionUtil.java

```diff
@@ -33,11 +38,17 @@
 public class ExpressionUtil {
   private static final Function<Object, Integer> HASH_FUNC =
       Transforms.bucket(Integer.MAX_VALUE).bind(Types.StringType.get());
+  private static final OffsetDateTime EPOCH = Instant.ofEpochSecond(0).atOffset(ZoneOffset.UTC);
+  private static final long FIVE_MINUTES_IN_MICROS = TimeUnit.MINUTES.toMicros(5);
+  private static final long THREE_DAYS_IN_HOURS = TimeUnit.DAYS.toHours(3);
+  private static final long NINETY_DAYS_IN_HOURS = TimeUnit.DAYS.toHours(90);
   private static final Pattern DATE = Pattern.compile("\\d\\d\\d\\d-\\d\\d-\\d\\d");
```

Comment:
regex could simpfly to "\\d{4}-\\d{2}-\\d{2}"

Reply:
Done.